cachi2/core/package_managers/bundler/scripts/lockfile_parser.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+
+require 'bundler'
+require 'json'
```

Comment:
I'm getting this error by using an image I build with the current version of the Dockerfile in this PR:
```
2024-09-03 17:59:53,529 ERROR The command "/venv/lib64/python3.9/site-packages/cachi2/core/package_managers/bundler/scripts/lockfile_parser.rb" failed
2024-09-03 17:59:53,529 ERROR STDERR:
<internal:/usr/share/rubygems/rubygems/core_ext/kernel_require.rb>:85:in `require': cannot load such file -- json (LoadError)
	from <internal:/usr/share/rubygems/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /venv/lib64/python3.9/site-packages/cachi2/core/package_managers/bundler/scripts/lockfile_parser.rb:4:in `<main>'
2024-09-03 17:59:53,529 ERROR PackageManagerError: Failed to parse /target/Gemfile.lock

```

Do we also need to install a `json` dependency?

Reply:
Ah, I think this is the problem:
` --setopt install_weak_deps=0 `

Reply:
`rubygems-json` is enough to pull in the rest of the strong dependencies AND to parse a lockfile.